### PR TITLE
Update funfuzz to work with lithium 0.2.0

### PR DESCRIPTION
**Pull request by nth10sd**
This seems to be the minimal set of changes necessary to make funfuzz work with Lithium 0.2.0 (currently in [PR #49 over there](https://github.com/MozillaSecurity/lithium/pull/49))

Some changes were adapted from one of [truber's commits](https://github.com/MozillaSecurity/domfuzz/commit/7e462aa842c161db87d29f05275b1d2d89ec3e1c) in domfuzz.

### CODE REVIEW

File: js/inspectShell.py [hunk 1]
```diff
@@ -10,20 +10,9 @@
 
 import os
 import platform
-import sys
 
-path0 = os.path.dirname(os.path.abspath(__file__))
-path1 = os.path.abspath(os.path.join(path0, os.pardir, 'util'))
-sys.path.append(path1)
 import subprocesses as sps
```
Comment (jschwartzentruber) [inline js/inspectShell.py]: Does this work without `sys.path` foo above?

Comment (nth10sd) [reply js/inspectShell.py]: Not sure. I wouldn't worry about it yet, there's still a bunch of sys.path stuff lying around that I'll be removing in a future PR anyway.

**Comment by nth10sd** [comment]
The tests for this latest commit will fail until merging after Lithium 0.2.0 PR 49 lands.

File: js/inspectShell.py (hunk 1 above)
Comment (nth10sd) [reply js/inspectShell.py]: Oh, wait, I misunderstood you. Yes, this was mistakenly removed, it wouldn't work without those lines.

**Pull request merged**